src/TimeZones.jl

```diff
@@ -90,12 +64,44 @@ include("local.jl")
 include("ranges.jl")
 include("discovery.jl")
 include("rounding.jl")
-include("parse.jl")
-include("deprecated.jl")
 
 # Required to support Julia `VERSION < v"1.9"`
 if !isdefined(Base, :get_extension)
     include("../ext/TimeZonesRecipesBaseExt.jl")
 end
 
+function update_dates_extension()
```

Comment:
Can you move this function into "parse.jl"?
```suggestion
function _apply_parsing_extensions()
```